Gemfile

```diff
@@ -5,14 +5,11 @@ source 'https://rubygems.org'
 # development dependencies will be added by default to the :development group.
 gemspec
 
-gem 'ransack', github: 'activerecord-hackery/ransack'
 gem 'ransack-mongoid', github: 'activerecord-hackery/ransack-mongoid'
 
 # Test app stuff
 
-gem 'rails', '~> 5.1'
-gem 'kaminari', '~> 1.0'
-gem 'kaminari-mongoid'
+gem 'rails'
```

Comment:
I would keep the version specification to make it explict the rails version we're using for testing.

Reply:
oh, i see. okay, i'll change it then